modules/auxiliary/gather/browser_sop_bypass.rb

```diff
@@ -0,0 +1,53 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpServer
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name'           => "Browser SOP Bypass",
```

Comment:
You need to be specific about the browser. It's unlikely one bug will affect IE in the same way it'll affect Opera. I realize you talk about it in #8977, but the module needs to give a hint as to what it actually affects.